test/integration.sh

```diff
@@ -50,6 +50,7 @@ token=$(curl --request POST \
   ./tmp/bin/thanos receive \
     --receive.hashrings-file=./test/config/hashrings.json \
     --receive.local-endpoint=127.0.0.1:10901 \
+    --receive.default-tenant-id="1610b0c3-c509-4592-a256-a1871353dbfa" \
```

Comment:
This is needed because otherwise, the first request will be failing while we test and the results will be below %100.